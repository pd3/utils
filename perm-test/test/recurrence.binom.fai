1	2	52	60	61

