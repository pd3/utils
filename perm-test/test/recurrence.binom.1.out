SEED	1587500664
NITER	1.000000e+05
# TEST:
#     - gene .. the region label
#     - nObserved .. number of hits observed in input data in this gene
#     - nSimFewer .. number of simulations with fewer hits in the gene than observed in the data
#     - nSimEqual .. as above, but with the same number of hits than observed
#     - nSimExceeded .. as above, but with the more hits than observed
# DIST:
#     - gene
#     - nSimHits .. the number of simulations with 0,1,2,etc. hits in the gene
# [1]TEST	[2]gene	[3]nObserved	[4]nSimFewer	[5]nSimEqual	[6]nSimExceeded
TEST	side1	2	74913	25087	0
TEST	side2	0	0	25087	74913
# [1]DIST	[2]gene	[3-]nSimHits (0,1,etc. hits)
DIST	side1	25013	49900	25087
DIST	side2	25087	49900	25013
